src/ui/pages/Menu/Menu.tsx

```diff
@@ -131,6 +134,21 @@ const Menu = () => {
     setShowConnections(selectedOption === SubMenuKey.Connections);
   }, [selectedOption]);
 
+  const handleOpenUrl = (key: SubMenuKey) => {
```

Comment:
I think you could be more specific and use `key: SubMenuKey.Crypto | SubMenuKey.Chat` here to stop it being called by other values and just hitting a blank `return`